CONTRIBUTING.md

```diff
@@ -0,0 +1,61 @@
+# Welcome to zium.app contributing guide <!-- omit in toc -->
```

Comment:
@toiletgranny do you want to add some info on how to contribute and what would be the flow for design contributions?

Reply:
I'm on it!